### PR TITLE
docs: About e2e prow jobs

### PR DESCRIPTION
**Pull request by navidshaikh**
## Description
List the two e2e prow jobs being run in the CI.

### CODE REVIEW

**Comment by dsimansk** [comment]
@navidshaikh looks good.

Maybe I'd add a single test example, for the convenience to copy/paste. Wdyt?
```
 $./test/local-e2e-tests.sh -run ^TestService$ 
```

File: test/README.md [hunk 1]
```diff
@@ -7,18 +7,18 @@ This directory contains e2e tests and testing docs:
...
 
-_By default `go test` will not run [the e2e tests](#running-e2e-tests-locally),
+_By default `go test` will not run [the e2e tests](#running-e2e-tests),
 which need [`-tags=e2e`](#running-end-to-end-tests) to be enabled._
 
-## Running e2e tests locally
+## Running e2e tests
 
-To run [the e2e tests](./e2e) , you need to have a
+To run [the e2e tests](./e2e) locally, you need to have a
```
Comment (dsimansk) [inline test/README.md:21]:
```suggestion
To run [the e2e tests](./e2e) locally, you need to have:
```

File: test/README.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ mode, use
 ```bash
 test/local-e2e-tests.sh -short
 ```
+### e2e prow jobs
```
Comment (dsimansk) [inline test/README.md:69]: `E2E Prow jobs` looks better? I can't decide. :)

Comment (navidshaikh) [reply test/README.md]: changed and updated all headings to match the style. :)

**Comment by navidshaikh** [comment]
> Maybe I'd add a single test example, for the convenience to copy/paste. Wdyt?

its [there](https://github.com/knative/client/pull/1250/files#diff-5de36acd90308dc62abf7855a686ee7052ffb6e762c756fd735fb0c9fbd9595dR56)

**Review by dsimansk — APPROVED** [review]
/lgtm

**Review by maximilien — APPROVED** [review]
Nice. Thanks for making that clear for all. Cheers 🍻 
/lgtm

**Pull request merged**